### PR TITLE
Changes related to issue #4 (LG G2 Sprint, LS980)

### PR DESCRIPTION
**Pull request by Mathnerd314**
This handles the overflow error in #4 and also a timeout error I came across.

You probably don't want to merge these commits as-is, I don't think they're portable.

### CODE REVIEW

File: lglaf.lua [hunk 1]
```diff
@@ -38,9 +38,9 @@ function lglaf.dissector(tvb, pinfo, tree)
     local endpoint = usb_endpoint().value
 
     -- Process only bulk packets from (EP 5) and to the device (EP 3)
-    if not ((endpoint == 0x85 or endpoint == 3) and transfer_type == 3) then
-        return 0
-    end
+    -- if not ((endpoint == 0x85 or endpoint == 3) and transfer_type == 3) then
+    --    return 0
+    -- end
```
Comment (Lekensteyn) [inline lglaf.lua]: Yes this is unfortunate, I had to do the same to test your capture (actually, the bulk transfer type check can stay).

Maybe I have to detect this earlier or turn it into a pref (or drop the EP number check).

**Comment by Lekensteyn** [comment]
The device name needs to be probed apparently to work on older devices. The overflow detection and close/open/read fallback is too ugly to get in ;-)

I'll keep it open in case other users encounter similar issues.

**Comment by Jookia** [comment]
This patch was needed for my friend to download a 4.5GB partition from their LG G4 H810. Without it, it timed out around the ~500MB mark.

**Comment by Mathnerd314** [comment]
Closing in favor of #60

**Pull request closed without merging**